preprod.healdata.org/manifest.json

```diff
@@ -20,8 +20,8 @@
     "metadata": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/metadata-service:1.8.2",
     "peregrine": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/peregrine:2022.08",
     "pidgin": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/pidgin:2022.08",
-    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:3.32.1",
-    "requestor": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/requestor:1.7.1",
+    "portal": "quay.io/cdis/data-portal:feat_workspace-registration",
+    "requestor": "quay.io/cdis/requestor:1.7.1",
```

Comment:
why change this to use Quay?

Reply:
No reason in particular. I think we used a quay build in the qa environment, and decided to move ahead with the same in preprod as well. But I changed it back, since it would not impact any of the features.

Reply:
We've had a ton of issues with Quay being unreliable, so we moved to ECR for production.